sdk/blockchain/blockchain.go

```diff
@@ -391,7 +389,7 @@ func (c *Subnet) InitializeProofOfAuthority(
 		if !errors.Is(err, validatormanager.ErrAlreadyInitialized) {
 			return evm.TransactionError(tx, err, "failure initializing poa validator manager")
 		}
-		ux.Logger.PrintToUser("Warning: the PoA contract is already initialized.")
+		log.Warn("the PoA contract is already initialized")
```

Comment:
i think this can be log.info. And we can make it:  the PoA contract is already initialized, skipping initializing Proof of Authority contract